src/oss/confluent.sh

```diff
@@ -623,6 +631,19 @@ command_exists() {
     exists "${command}" "commands"
 }
 
+dynamic_cli_command_exists() {
+    local command="${1}"
+    exists "${command}" "dynamic_cli_commands"
+}
+
+dynamic_cli_file_exists(){
+    local command="${1}"
+    if [ -e "${command}-cli" ] ; then
```

Comment:
For the sake of uniformity, but also because it's a better check you'll want to use 
`if [[ -f "${confluent_bin}/${command}-cli" ]] ; then`

(also notice that again the path should be absolute, not relative)